crates/kitsune_p2p/kitsune_p2p/src/gossip/simple_bloom.rs

```diff
@@ -63,7 +63,7 @@ pub enum MetaOpKey {
 #[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub enum MetaOpData {
     /// data chunk type
-    Op(Arc<KitsuneOpHash>, Vec<u8>),
+    Op(Arc<KitsuneOpHash>, KOp),
```

Comment:
Why are we still wasting cycles maintaining simple_bloom, we're not expecting anyone to use it. I say we delete it.